virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,43 @@
+"""
+libvirt related misc functions
+"""
+import logging
+
+from avocado.core import exceptions
+
+from virttest import libvirt_version
+
+
+def is_libvirt_feature_supported(params, ignore_error=False):
+    """
+    Check whether the function is supported in this libvirt version by comparing
+    the installed libvirt version
+
+    :param params: Dictionary with the test parameters
+    :param ignore_error: Whether to raise an exception
+    :raise: When ignore_error is set False, raise TestCancel if the feature is
```

Comment:
I assume 'TestCancel" means that execption since TestCancel apply tp-libvirt project scope only